src/components/docs-version-switcher/index.tsx

```diff
@@ -10,30 +10,24 @@ import { getVersionFromPath } from 'lib/get-version-from-path'
 import { removeVersionFromPath } from 'lib/remove-version-from-path'
 import { useCurrentProduct } from 'contexts'
 import useCurrentPath from 'hooks/use-current-path'
-import DropdownDisclosure, {
-	DropdownDisclosureLabelItem,
-	DropdownDisclosureAnchorItem,
-} from 'components/dropdown-disclosure'
+import VersionSwitcher, {
+	VersionSwitcherOption,
+} from 'components/version-switcher'
 import { DocsVersionSwitcherOption, DocsVersionSwitcherProps } from './types'
-import s from './docs-version-switcher.module.css'
 
 const IS_DEV = process.env.NODE_ENV !== 'production'
 
+/**
+ * Render a docs version switcher directly from loaded `VersionSelectItem`s,
+ * wrapping a generic `VersionSwitcher` in docs-related logic.
+ */
 const DocsVersionSwitcher = ({
 	options,
 	projectName,
 }: DocsVersionSwitcherProps) => {
 	const currentProduct = useCurrentProduct() as ProductWithCurrentRootDocsPath
 	const currentPath = useCurrentPath({ excludeHash: true, excludeSearch: true })
 
-	// Don't do anything if there aren't any options
-	if (!options || options.length === 0) {
```

Comment:
Rendering `null` when there are no `options` is now handled by the generic `VersionSwitcher`.